Jormungand.DiscordReader-Mouth/src/main/java/Jormungandr.java

```diff
@@ -54,7 +56,7 @@ public void onGuildReady(GuildReadyEvent event) {
         JorgReadyEvent jre = new JorgReadyEvent(event);
         System.out.println(gson.toJson(jre));
     }
-    //TODO for ALL: move this all
```

Comment:
The events are still in the main class instead of a seperate class that contains all the listeners, see issues #1

Reply:
Does from `onGuildMemberUpdateNickname` to `onMessageReceived` need to be moved out? Specific clarification on 'all'.